### ISSUE TITLE
syslog size for public peer

### ISSUE THREAD

**Issue by d47081** [yggdrasil-network/yggdrasil-go#1108]
hello there!

I found that my public peer server (yggdrasil 0.5) starting to generate massive `/var/log/syslog` files with following content

```
Nov 19 00:01:23 yggdrasil[387592]: 2023/11/19 00:01:23 Connected inbound: 201:23b4:991a:634d:8359:4521:5576:15b7@94.140.114.241:39284, source 94.140.114.241:4708
Nov 19 00:01:23 yggdrasil[387592]: 2023/11/19 00:01:23 Disconnected inbound: 201:23b4:991a:634d:8359:4521:5576:15b7@94.140.114.241:39284, source 94.140.114.241:4708; error: ErrBadKey: Expected KEY, Found KEY
...
```

is there any way how to reduce it with debug level or that's bad idea?
I'm using default deb package with systemd without any changes in auto-generated config

Just size of logs grow up to 1 Gb, I think, that's issue for VPS

**Comment by majestrate** [comment]
`logrotate` package should be able to handle this.

**Comment by majestrate** [comment]
note: ygg 0.5 got rid of the DHT spanning tree so the concept of a root node is no more, it's replaced with a "core" set of routers which are the most well connected.

**Comment by basilgello** [comment]
It is the regression (hopefully) fixed in v0.5.4. We need to wait until all public peers upgrade to 0.5.4

**Comment by Saiv46** [comment]
@d47081 You can close this issue? I hope you found where to put your option. Also people upgraded to 0.5.4
